libs/execution_context/ebpf_maps.c

```diff
@@ -875,6 +908,13 @@ ebpf_map_create(
         goto Exit;
     }
 
+    if (inner_map_handle != (ebpf_handle_t)-1) {
+        // Convert value handle to an object pointer.
+        result = ebpf_reference_object_by_handle(inner_map_handle, EBPF_OBJECT_MAP, &inner_map_template_object);
```

Comment:
The pattern so far has been to convert all handle -> object references in ebpf_core.c.

Reply:
Based on an offline discussion, this is an intentional move away from having all handle -> object conversion in ebpf_core.c and towards having the map specific logic in ebpf_map.c.